res/xml/number_row.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <row height="0.75">
-    <key key0="1"/>
-    <key key0="2"/>
-    <key key0="3"/>
-    <key key0="4"/>
-    <key key0="5"/>
-    <key key0="6"/>
-    <key key0="7"/>
-    <key key0="8"/>
-    <key key0="9"/>
-    <key key0="0"/>
+    <key key0="1" sw="!"/>
```

Comment:
The `sw` direction might be hard to hit on this key as it's close to the edge of the screen. The `se` direction doesn't go toward the edge of the screen and feels more natural for the thumb.
What do you think of using the `se` direction for the symbols on keys 1 to 5 ?

Reply:
I'm actually using the `s` direction in my current embedded-number-row layouts (the compact BQN layout also squeezing `` `~|\ `` on downwards swipes in its number row, in addition to the BQN upwards swipes). So I'm fine with any change here.

Reply:
Switched 1-5 to `se`; works out nicely with the BQN layout ([current look of it](https://github.com/dzaima/Unexpected-Keyboard/commit/4891cd69963ebbd1cd36762a4d4abfbef2721fac)).

Reply:
Thanks for the change! I think `!()` are quite easy to type, which is what matters the most. The BQN layout is definitely an advanced layout, perhaps seeking consistency with the builtin number row is not needed ?

With the `s` direction, I would be afraid that the symbols are too close together and look messy.